core/src/main/java/nl/nn/adapterframework/lifecycle/IbisApplicationInitializer.java

```diff
@@ -63,6 +67,20 @@ protected WebApplicationContext createWebApplicationContext(ServletContext servl
 		return applicationContext;
 	}
 
+	/**
+	 * Loads webApplicationContext and files specified by the SPRING.WAC.CONFIG.LOCATIONS
+	 * property in AppConstants.properties
+	 * 
+	 * @param classLoader to use in order to find and validate the Spring Configuration files
+	 * @return A String array containing all files to use.
+	 */
+	private String[] getSpringConfigurationFiles(ClassLoader classLoader) {
+		List<String> baseConfigLocations = new ArrayList<>();
+		baseConfigLocations.add(XmlWebApplicationContext.CLASSPATH_URL_PREFIX + "/webApplicationContext.xml");
+		return IbisApplicationContext.getSpringConfigurationFiles(classLoader, baseConfigLocations,
+				"SPRING.WAC.CONFIG.LOCATIONS", XmlWebApplicationContext.CLASSPATH_URL_PREFIX, log);
```

Comment:
Je kan beter `ResourceUtils.CLASSPATH_URL_PREFIX` gebruiken, daar komt hij oorsprokelijk vandaan.
En de classpathUrlPrefix is overal hetzelfde dus die kan weg :) scheelt weer een argument

Reply:
Done